### PR TITLE
 Require hard deps instead of relying on soft ones from fw bundle 

### PR DESCRIPTION
**Pull request by Lctrs**
Also :
- Move files into `src/` and tests files into `tests/`.
- Sort packages by alphabetical orders

Supersed #204.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -2,4 +2,4 @@
 /.gitignore export-ignore
 /.travis.yml export-ignore
 /phpunit.xml.dist export-ignore
-/Tests export-ignore
+/tests export-ignore
```
Comment (Seldaek) [inline .gitattributes]: This is incorrect

Comment (Seldaek) [reply .gitattributes]: Ah now I see you renamed all the things below.. never mind me

**Comment by Seldaek** [comment]
Any objections @romainneutron ?

**Comment by Lctrs** [comment]
Rebased. Failure is unrelated.

**Pull request closed without merging**